src/components/Header/NavBar/Dialog/Dialog.js

```diff
@@ -106,58 +106,42 @@ const FormDialog = memo(({ show, handleClose, address, account, amount, amountAi
 							},
 						};
 					});
-					console.log({ total: msg, total_amount });
-
+					payload = args({ msg, type: typeSubmit, fromAddress: address, totalAmount: total_amount });
```

Comment:
I think to be safe, the total_amount here should be parsed toFixed(0) so that it is 100% an integer in string format